go/processor/floria/state_contract.go

```diff
@@ -0,0 +1,246 @@
+
+var (
+	setBalanceMethodID []byte
+	copyCodeMethodID   []byte
+	swapCodeMethodID   []byte
+	setStorageMethodID []byte
+	incNonceMethodID   []byte
+)
+
+func init() {
+	abi, err := abi.JSON(strings.NewReader(stateContractABI))
+	if err != nil {
+		panic(fmt.Errorf("failed to parse stateContractABI: %w", err))
+	}
+
+	for name, constID := range map[string]*[]byte{
+		"setBalance": &setBalanceMethodID,
+		"copyCode":   &copyCodeMethodID,
+		"swapCode":   &swapCodeMethodID,
+		"setStorage": &setStorageMethodID,
+		"incNonce":   &incNonceMethodID,
+	} {
+		method, exist := abi.Methods[name]
+		if !exist {
+			panic("unknown EvmWriter method")
+		}
+
+		*constID = make([]byte, len(method.ID))
+		copy(*constID, method.ID)
+	}
+}
+
+var ErrExecutionReverted = fmt.Errorf("execution reverted")
+var ErrOutOfGas = fmt.Errorf("out of gas")
+
+// handleStateContract is a reworked version of the original function from the Opera client.
+// It is used to handle epochs and allows to set balance, copy code, swap code, set storage, and increment nonce.
+// Source: https://github.com/Fantom-foundation/Sonic/blob/main/opera/contracts/evmwriter/evm_writer.go#L24
+func handleStateContract(
```

Comment:
no need to change it now, but honestly, for function interface I think is better to have them in one line or multiline if over 80 columns. I agree that for test cases a liner per parameter is easier to read. But this to me seems clunky. it is just a personal opinion.

Reply:
It just don't fit in the side-by-side Github diff. And it turns quite difficult to read. That is it.